mmocr/models/textrecog/layers/attention_aster_layer.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class AttentionLSTM(nn.Module):
```

Comment:
The same as aster_decoder, there is already a corresponding implementation in MMOCR